app/vmctl/flags.go

```diff
@@ -119,6 +120,10 @@ var (
 			Name:  vmDisableProgressBar,
 			Usage: "Whether to disable progress bar per each worker during the import.",
 		},
+		&cli.BoolFlag{
+			Name:  vmInsecureSkipVerify,
+			Usage: "Whether to skip TLS certificate verification.",
```

Comment:
The description should mention explcitily requests to which target will skip verification: is it destination, or source, or both?